assets/js/googlesitekit/components-gm2/ProgressBar.js

```diff
@@ -21,17 +21,50 @@
  */
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
+import {
+	BREAKPOINT_DESKTOP,
+	BREAKPOINT_SMALL,
+	BREAKPOINT_TABLET,
+	BREAKPOINT_XLARGE,
+	useBreakpoint,
+} from '../../hooks/useBreakpoint';
 
 export default function ProgressBar( {
 	className,
 	small,
 	compress,
 	indeterminate,
 	height,
+	smallHeight,
+	tabletHeight,
+	desktopHeight,
 	progress,
 } ) {
+	const breakpoint = useBreakpoint();
+
+	let progressBarHeight = height;
+
+	if ( BREAKPOINT_SMALL === breakpoint && smallHeight ) {
+		progressBarHeight = smallHeight;
+	}
+
+	if ( BREAKPOINT_TABLET === breakpoint && tabletHeight ) {
```

Comment:
Seeing as these are a set of mutually exclusive conditions, please can we use `else if` so that our code structure semantically follows the relationship between the conditions?